test/perf/perf_test.go

```diff
@@ -95,3 +138,105 @@ func initLogger() (logr.Logger, *os.File, error) {
+
+	hostOperatorPod, err := hostAwait.GetHostOperatorPod()
+	require.NoError(t, err)
+	memberOperatorPod, err := memberAwait.GetMemberOperatorPod()
+	require.NoError(t, err)
+
+	for b := 0; b < config.GetUserCount()/config.GetUserBatchSize(); b++ { // provisioning users by batch of `config.GetUserBatchSize()` (eg: 100)
+
+		signups := make([]toolchainv1alpha1.UserSignup, config.GetUserBatchSize())
+		start := time.Now()
+
+		for i := 0; i < config.GetUserBatchSize(); i++ {
+			n := (b*config.GetUserBatchSize() + i)
+			name := fmt.Sprintf("multiple-signup-testuser-%d", n)
+			// check if there is already a MUR with the expected name, in which case, continue with the next one
+			mur := toolchainv1alpha1.MasterUserRecord{}
+			if err := hostAwait.Client.Get(context.TODO(), types.NamespacedName{Namespace: hostAwait.Namespace, Name: name}, &mur); err == nil {
+				t.Logf("no need to create a UserSignup for '%s', the MasterUserRecord resource already exists", name)
+				// skip this one, it already exists
+				continue
+			}
+			// Create an approved UserSignup resource
+			userSignup := NewUserSignup(t, hostAwait, memberAwait, name, fmt.Sprintf("multiple-signup-testuser-%d@test.com", n), false)
+			userSignup.Spec.Approved = true
+			err := hostAwait.FrameworkClient.Create(context.TODO(), userSignup, CleanupOptions(ctx))
+			hostAwait.T.Logf("created usersignup with username: '%s' and resource name: '%s'", userSignup.Spec.Username, userSignup.Name)
+			require.NoError(t, err)
+			signups[i] = *userSignup
+		}
+		err := hostAwait.WaitUntilMetricHasValueOrLess(hostMetricsRoute.Status.Ingress[0].Host, "workqueue_depth", "name", "masteruserrecord-controller", 0)
+		require.NoError(t, err)
+
+		for _, signup := range signups {
+			mur, err := hostAwait.WaitForMasterUserRecord(signup.Spec.Username, UntilMasterUserRecordHasCondition(Provisioned()))
+			require.NoError(t, err)
+			// now, run a pod (with the `sleep 7200` command in each namespace)
+			userAccount, err := memberAwait.WaitForUserAccount(mur.Name,
+				wait.UntilUserAccountHasConditions(Provisioned()))
+			require.NoError(t, err)
+			for _, templateRef := range userAccount.Spec.NSTemplateSet.Namespaces {
+				ns, err := memberAwait.WaitForNamespace(mur.Name, templateRef.TemplateRef)
+				require.NoError(t, err)
+				if ns.Labels["toolchain.dev.openshift.com/type"] != "stage" {
+					// skip pod creation if the namespace is not "stage", otherwise, we may run out of capacity of pods on the nodes
+					continue
+				}
+				pod := corev1.Pod{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: ns.Name,
+						Name:      "sleep",
+						Labels: map[string]string{ // just so we can list them from the CLI if needed
+							"toolchain.dev.openshift.com/type": "test",
+						},
+					},
+					Spec: corev1.PodSpec{
+						Containers: []corev1.Container{
+							{
+								Name:    "sleep",
+								Image:   "busybox",
+								Command: []string{"sleep", "36000"}, // 10 hours
```

Comment:
Let's set it to 8hours which is the timeout for the idler
```suggestion
								Command: []string{"sleep", "28800"}, // 8 hours - same as for idler
```